src/build_prompt.py

```diff
@@ -0,0 +1,45 @@
+from tokenizer import get_token_from_string
+from tokenizer import get_string_from_tokens
+
+def get_prompt(
+    query: str,
+    chunks: dict,) -> str:
+    """From a question and relevant documents, build the prompt to ask the LLM.
+
+    Args:
+        query (str): Question to ask the LLM.
+        chunks (dict): Dictionary containing the text of the chunks as values.
+
+    Returns:
+        str: The completed prompt, combined from the template, query and chunks.
+    """
+
+    context = ""
+    for key in chunks:
```

Comment:
In a dictionary the keys are not in a defined sort order. Then the most relevant chunk may be at the end. Please rather use a list.